roles/cdi/tasks/deprovision.yml

```diff
@@ -0,0 +1,6 @@
+---
+- set_fact:
+    storage_demo_node_hostname: "nohost"
+
+- name: Delete storage-demo Resources
+  command: kubectl delete deployment cdi-deployment
```

Comment:
Would it be more symmetric to generate the yaml file and use kubectl delete -f ?